ebpf/event_common.go

```diff
@@ -98,6 +99,16 @@ type ConnectionStats struct {
 	IPTranslation *netlink.IPTranslation `json:"conntrack"`
 }
 
+// SourceAddr returns the source address in the Address abstraction
+func (c ConnectionStats) SourceAddr() Address {
+	return c.Source.(Address)
```

Comment:
Should you handle nil here?

Reply:
I had initially added it, but the amount of code needed to check it everywhere affected code readibility. This cast is safe to do everywhere except process-agent which does a string cast (which has comments on why).

In the next sprint or two, we'll rip out easy json and plugin protobuf (with the option to output as json as well for debugging) which will clean all this up